manifest.yml

```diff
@@ -8,6 +8,8 @@ inputs:
     description: Authentication token for Sentry
   - name: sentryRelease
     description: The release ID (a.k.a version)
+  - name: sentryRepository
+    description: Override the name of the target Sentry repository
```

Comment:
@lewisjared  Same as above, description to "The name of the target Sentry repository"